exercises/palindrome-products/palindrome_products_test.go

```diff
@@ -37,29 +38,44 @@ var testData = []struct {
 		""},
 	{4, 10, Product{}, Product{}, "No palindromes"},
 	{10, 4, Product{}, Product{}, "fmin > fmax"},
-	/* bonus curiosities.  (can a negative number be a palindrome?
-	// most say no
-	{-99, -10, Product{}, Product{}, "Negative limits"},
-	// but you can still get non-negative products from negative factors.
+}
+
+// Bonus curiosities. Can a negative number be a palindrome? Most say no.
+var bonusData = []struct {
+	fmin, fmax int
+	pmin, pmax Product
+	errPrefix  string
+}{
+	// The following two test cases have the same input, but different expectations. Uncomment just one or the other.
+
+	/* Here you can test that you can reach the limit of the largest palindrome made of two 2-digit numbers.
+	{-99, -10, Product{}, Product{}, "Negative limits"}, */
+
+	// You can still get non-negative products from negative factors.
 	{-99, -10,
 		Product{121, [][2]int{{-11, -11}}},
 		Product{9009, [][2]int{{-99, -91}}},
 		""},
```

Comment:
yes :( I have all the go toolings installed in my IDE and they are activated every time i save. To be more sure I also just used `go fmt` on the file from the command line and the result is the same :(